packages/compiler/src/ast/index.ts

```diff
@@ -25,7 +25,7 @@ export {
   visitChildren,
   type NodeCallback,
 } from "../core/parser.js";
-
+export { createSourceLoader } from "../core/source-loader.js";
```

Comment:
Should we make this an experimental export so we can make breaking changes if needed?

Reply:
I think its probably fine as this been there forever but lets just be safe, moved